CMakeLists.txt

```diff
@@ -95,9 +95,17 @@ configure_file(
   ${CMAKE_CURRENT_BINARY_DIR}/revision.h
 )
 
+SET( HEADER_PATH include/assimp )
+
+configure_file(
+  ${HEADER_PATH}/config.h.in
+  ${ASSIMP_INCLUDE_INSTALL_DIR}/assimp/config.h
```

Comment:
@kimkulling looks like this is causing issues on the build server - maybe `ASSIMP_INCLUDE_INSTALL_DIR` is not defined at this point?